x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -37,12 +37,13 @@ appender.audit_rolling.layout.pattern = {\
                 %varsNotEmpty{, "transport.profile":"%enc{%map{transport.profile}}{JSON}"}\
                 %varsNotEmpty{, "rule":"%enc{%map{rule}}{JSON}"}\
                 %varsNotEmpty{, "event.category":"%enc{%map{event.category}}{JSON}"}\
+                %varsNotEmpty{, "config_change":%map{config_change}}\
```

Comment:
The new field that contains the configuration taking effect, as an un-escaped JSON object, for records with `event.type: "security_config_change"`.